<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="15703" started-at="2019-05-03T12:47:28Z" finished-at="2019-05-03T12:47:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="15703" started-at="2019-05-03T12:47:28Z" finished-at="2019-05-03T12:47:44Z">
      <class name="projectName.frontend.test.t_TravelSection">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7]" name="beforeSuite" is-config="true" duration-ms="176" started-at="2019-05-03T19:47:28Z" finished-at="2019-05-03T19:47:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7]" name="setUp" is-config="true" duration-ms="5288" started-at="2019-05-03T19:47:28Z" finished-at="2019-05-03T19:47:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void projectName.frontend.test.t_TravelSection.TC001_Verify_Travel_Section() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TC001_Verify_Travel_Section()[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7]" name="TC001_Verify_Travel_Section" duration-ms="9113" started-at="2019-05-03T19:47:34Z" finished-at="2019-05-03T19:47:43Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[At least 3 cards are being not displayed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: At least 3 cards are being not displayed
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at core.action.BaseAction.assertFail(BaseAction.java:642)
	at core.action.BaseAction.assertResult(BaseAction.java:634)
	at projectName.frontend.test.t_TravelSection.TC001_Verify_Travel_Section(t_TravelSection.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_Verify_Travel_Section -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7]" name="tearDown" is-config="true" duration-ms="1291" started-at="2019-05-03T19:47:43Z" finished-at="2019-05-03T19:47:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TC001_Verify_Travel_Section status=FAILURE method=t_TravelSection.TC001_Verify_Travel_Section()[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:projectName.frontend.test.t_TravelSection@57fffcd7]" name="afterSuite" is-config="true" duration-ms="217" started-at="2019-05-03T19:47:44Z" finished-at="2019-05-03T19:47:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- projectName.frontend.test.t_TravelSection -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
